src/main/drivers/mco.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "platform.h"
+
+#ifdef USE_MCO
+
+#include "drivers/io.h"
+#include "pg/mco.h"
+
+void mcoInit(const mcoConfig_t *mcoConfig)
+{
+    // Only configure MCO2 with PLLI2SCLK as source for now.
+    // Other MCO1 and other sources can easily be added.
+    // For all F4 and F7 varianets, MCO1 is on PA8 and MCO2 is on PC9.
+
+    if (mcoConfig->ioTag[1] == IO_TAG(PC9) && mcoConfig->enabled[1]) {
```

Comment:
I don't think this will work in this way - `mcoConfig->ioTag[1]` will always be `NONE`. We should just remove the `mcoConfig->ioTag[1] == IO_TAG(PC9)` check and hardcode for `PC9`, since this is the only choice.

Reply:
The check is intended for those who fiddle with `resource` command to see what other pins would do. Similar check is present for UART and SPI.

Oh, do we want the MCO resource to appear in `resource list` without being listed in `resource`?

Reply:
EDIT
Yeah, I think USB does the same thing.

Reply:
Variable name ... `mco2_on_pc9 = ENABLE,DISABLE`. Is this fine for everybody?

Reply:
Yes to listing it in resources without making it assignable - we'll need this facility for other things as well.

And just make the parameter use on / off, for consistency and space economy.